projects/2019_12_14_pkt-whitepaper-website.md

```diff
@@ -372,3 +372,589 @@ The applicant understands that the Network Steward is not a legal entity and no
+Button: PKT Roadmap
+Button: PKT Origin Story
+
+#PKT Roadmap 
+There are four distinct stages of evolution which the PKT project will undergo. These stages may occur out of order at times, but the project will be viable in every state, just as a caterpillar is a viable creature before it becomes a butterfly.
+
+##The PKT Roadmap is:
+Build a VPN market where anyone can participate and the minimum price is free
+Support low cost Internet sharing hardware that allows people to earn PKT using their extra bandwidth
+Create a bandwidth market where there are virtual ISPs to do the network engineering so that building a mesh network is easy
+Support high performance networking hardware which automatically sells its bandwidth into the decentralized bandwidth market
+
+##Stage One, VPN Marketplace
+  
+Price will be decided by auction with a minimum bid of zero. In order to stay decentralized and achieve low enough cost of transactions to enable true micropayments, VPNs will be purchased using PKT, a cryptocurrency which is mined by performing encryption operations and expending Internet bandwidth. Anyone who wants to use a VPN, create their own, or mine PKT will easily be able to do so.
+
+##Stage Two, Support Internet Sharing
+  
+Once there are approximately one million installations of the VPN client application, PKT will begin supporting third party operators who offer internet sharing technology and hardware for home or business users. This technology and hardware will allow people to earn PKT with their unused internet bandwidth and like the VPN market, it will have a minimum bid of zero. Since it will be the VPN client application that connects to the Internet sharing hardware, there will be no privacy risk nor legal risk because all data will be sent through the VPN.
+
+##Stage Three, Tokenized Bandwidth Leases
+
+By tokenizing bandwidth leases Internet sharing hardware will be able to connect to each other and virtual ISPs will be able to observe the network, buy, sell and trade, relevant leases, and plan paths through it. This will make local mesh networking easy by moving the network engineering component to the virtual ISP layer.
+  
+##Stage Four, Support Next Gen Networking Hardware.
+  
+PKT will begin to support next generation networking hardware. The compact source routing central in the cjdns protocol will not only set new records for router performance/cost ratio, but will also allow for hundreds or even thousands of virtual ISPs to send data through a single device, each according to their own routing policy. Since PKT is mined by performing encryption operations, hardware devices capable of efficiently mining PKT will also be capable of efficiently encrypting network data. The ASICs which will emerge to mine PKT will be what finally enables the whole Internet to be encrypted.
```

Comment:
>the whole Internet to be encrypted

More precise language here. Like "all internet traffic to be encrypted" to emphasize transit traffic, or rest encryption, etc. whatever you intend to mean.

Reply:
I like the  ""enables the whole Internet to be encrypted" but I also agree that it needs to be qualified in terms of what  the encryption  mechanism  is. If all websites are  HTTPS, then the entire Internet is encrypted.

But PKT is better because  its end to end AND encrypted P2P in transit, making it more like TOR than like the traditional Internet.